src/Core/Components/DataGrid/Columns/PropertyColumn.cs

```diff
@@ -26,12 +27,12 @@ public class PropertyColumn<TGridItem, TProp> : ColumnBase<TGridItem>, IBindable
 	/// </summary>
 	[Parameter, EditorRequired] public Expression<Func<TGridItem, TProp>> Property { get; set; } = default!;
```

Comment:
Same here, why an Expression and not a Func?

Reply:
Same reason